certbot-dns-cloudflare/certbot_dns_cloudflare/_internal/dns_cloudflare.py

```diff
@@ -194,7 +202,10 @@ def _find_zone_id(self, domain):
 
         raise errors.PluginError('Unable to determine zone_id for {0} using zone names: {1}. '
                                  'Please confirm that the domain name has been entered correctly '
-                                 'and is already associated with the supplied Cloudflare account.'
+                                 'and is already associated with the supplied Cloudflare account. '
+                                 'If you\'re using Cloudflare Tokens, your Token needs '
+                                 '"Zone:DNS:Edit" and "Zone:Zone:Read" permissions for all zones '
+                                 'in your account.'
```

Comment:
I think we should remove this new info about Cloudflare token permissions. As I said in another comment, I think we should include the last raised error message here and I think that including both that and this new info about token permissions is too wordy.

I'd personally prefer including the specific error now and if we're getting (m)any bug reports about people hitting this case and being unsure what permissions to use on their token, we can maybe update things here.

What do you think?